### PR TITLE
Fixed transmission banner not showing on pass finalisation, and pre populated value

### PR DESCRIPTION
**Pull request by edwardwilson**
## Description
Fixes the transmission banner not showing on first selection in Mod 1
MES-5003 

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/pass-finalisation/cat-a-mod1/pass-finalisation.cat-a-mod1.page.html [hunk 1]
```diff
@@ -24,7 +24,6 @@
 
         <transmission
           [formGroup]="form"
-          [transmission]="pageState.transmission$ | async"
           (transmissionChange)="transmissionChanged($event)">
         </transmission>
```
Comment (rossfauldsbjss) [inline src/pages/pass-finalisation/cat-a-mod1/pass-finalisation.cat-a-mod1.page.html:29]: This feels like a bit of a hack. There must be a better way than deliberately not passing in state.

Comment (edwardwilson) [reply src/pages/pass-finalisation/cat-a-mod1/pass-finalisation.cat-a-mod1.page.html:29]: I think this is the most appropiate way, it's a one directional binding

Comment (rossfauldsbjss) [reply src/pages/pass-finalisation/cat-a-mod1/pass-finalisation.cat-a-mod1.page.html]: So - from the description there is no transmission for a cat-a-mod1.  So why is there a transmission component on the page.  If there is no transmission and yet the transmission component is responsible for displaying a banner this definitely doesn't seem right.  The banner display should be separated out from the component and displayed conditionally on the page.

Comment (rossfauldsbjss) [reply src/pages/pass-finalisation/cat-a-mod1/pass-finalisation.cat-a-mod1.page.html:29]: Removed objection - old title made me think there was no transmission on this test.

**Review by rossfauldsbjss — APPROVED** [review]
looks good

**Review by appleabs — APPROVED** [review]

**Pull request merged**